client/src/app/views/form.coffee

```diff
@@ -0,0 +1,37 @@
+$ = require('jquery')
+Marionette = require('backbone.marionette')
+template = require('../templates/form.jade')
+errorMessage = require('../templates/error.jade')
+SiteModel = require('../models/site')
+
+module.exports = Marionette.ItemView.extend(
+  model: new SiteModel()
+  template: template
+  
+  events:
+    'submit form' : 'enterSite'
+  
+  ui:
+    url: '#url'
+    email: '#email'
```

Comment:
This stuff shouldn't be ID's.  There is nothing truly unique and or singular in nature about them so they should use CSS classes not IDs